.circleci/config.yml

```diff
@@ -29,6 +29,20 @@ main_branches_filter: &main_branches_filter
         - staging
         - master
 
+commands:
+  setup_node12:
+    description: Setup Node 12
+    steps:
+      - run:
+          command: |
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v12
+            npm install
+            echo 'export NVM_DIR="/opt/circleci/.nvm"' >> $BASH_ENV
```

Comment:
this is instead of aliasing v12 to default?

Reply:
Not sure what you mean

Reply:
me too

Reply:
I thought that `nvm alias default 12` might do a similar work